watcher/watcher.go

```diff
@@ -1314,6 +1317,17 @@ func (w *Watcher) EnsureCorrectRooms() error {
 	timeoutDur := time.Duration(timeoutSec) * time.Second
 	willTimeoutAt := time.Now().Add(timeoutDur)
 
+	// We need to make sure that at least one goroutine is going to be used for the update of the pods,
+	// otherwise the operation will timeout.
+	goroutinePoolSize := w.Config.GetInt("watcher.goroutinePoolSize")
+	if goroutinePoolSize == 0 {
+		logger.Warn("watcher.goroutinePoolSize was 0, assigning it to 1")
+		goroutinePoolSize = 1
+	}
+
+	// TODO, FIXME(lhahn): remove this.
+	timeoutSec = 120
```

Comment:
Any reason why this value is being forced here? Also, I'm not sure if it does have any effect since `timeDur` is already set.

Reply:
Oh, this should be removed, I'll change that.